<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="distorted_camera">
    <link name="world">
      <pose>0 0 0 0 1.57 0</pose>
      <gravity>false</gravity>
      <static>true</static>
      <visual name="visual">
        <geometry>
          <box>
            <size>.1 .1 .1</size>
          </box>
        </geometry>
      </visual>
      <sensor name="camera" type="camera">
        <camera>
          <horizontal_fov>2.892</horizontal_fov>
          <image>
            <width>1280</width>
            <height>720</height>
          </image>
          <clip>
            <near>.2</near>
            <far>200</far>
          </clip>
          <distortion>
            <k1>-0.25</k1>
            <k2>0.12</k2>
            <k3>0.0</k3>
            <p1>-0.00028</p1>
            <p2>-0.00005</p2>
            <center>.5 .5</center>
          </distortion>
        </camera>
        <always_on>1</always_on>
        <update_rate>30</update_rate>
        <visualize>1</visualize>
      </sensor>
    </link>
  </model>
</sdf>
